numpy/_build_utils/src/apple_sgemv_fix.c

```diff
@@ -66,12 +69,21 @@ static int AVX_and_10_9 = 0;
 /* Dynamic check for AVX support
  * __builtin_cpu_supports("avx") is available in gcc 4.8,
  * but clang and icc do not currently support it. */
-#define cpu_supports_avx()\
-(system("sysctl -n machdep.cpu.features | grep -q AVX") == 0)
-
+#define cpu_supports_avx() ({\
```

Comment:
Any reason not just to use a `static inline cpu_supports_avx(void)` function here? This syntax is not standard.

Reply:
Hey Eric,
Yeah I made a mistake with that, I was a bit too focused on making it a macro haha. Let me make some changes and merge it again.
Thanks for the input,
Bran

P.S please reject it.